mech/__main__.py

```diff
@@ -46,6 +57,18 @@ def get_vm():
     exit()
 
 
+def get_vm_user():
```

Comment:
I don't like having the user have to set up a user. It should be optional in the mech file and only if the user wants to specify it himself in the JSON of the mechfile, like Sublime Text config files.